# Temporary comment out version check: Version mismatch between generated code and library headers

diff -Naur fastnetmon-1.2.0+git20220420.orig/src/simple_packet_capnp/simple_packet.capnp.h fastnetmon-1.2.0+git20220420/src/simple_packet_capnp/simple_packet.capnp.h
--- fastnetmon-1.2.0+git20220420.orig/src/simple_packet_capnp/simple_packet.capnp.h	2022-04-20 08:37:46.024989661 +0200
+++ fastnetmon-1.2.0+git20220420/src/simple_packet_capnp/simple_packet.capnp.h	2022-04-20 08:49:49.044341889 +0200
@@ -6,9 +6,6 @@
 #include <capnp/generated-header-support.h>
 #include <kj/windows-sanity.h>
 
-#if CAPNP_VERSION != 8000
-#error "Version mismatch between generated code and library headers.  You must use the same version of the Cap'n Proto compiler and library."
-#endif
 
 
 namespace capnp {
